src/section/shared_state.rs

```diff
@@ -403,8 +403,6 @@ impl SharedState {
         // - `[]`
         // - `[VoteTheirKey]`
         // - `[VoteTheirKey, VoteTheirKnowledge]`
-        // - `[VoteSendNeighbourInfo]`
-        // - `[VoteSendNeighbourInfo, VoteTheirKnowledge]`
```

Comment:
Better edit these comments to replace `VoteSendNeighbourInfo` -> `SendNeighbourInfo` instead of deleting them.